aced_submission/grip_load.py

```diff
@@ -0,0 +1,227 @@
+import os
+import orjson
+import requests
+from typing import List
+
+
+def bulk_add(graph_name: str, project_id: str, directory_path: str, output: dict, access_token: str) -> List[dict]:
+    """Loads a directory of .ndjson or .gz files to grip.
+        Args:
+                graph_name: The name of the graph
+                project_id: the Gen3 program-project to be used
+                directory_path: the file path to the directory that contains the FHIR files
+
+        TODO: implement FHIR schema in grip
+                    so that edges that don't validate in graph are rejected
+    """
+    response_json= []
+
+    # List graphs and check to see if graph name is amoung the graphs listed
+    exists = graph_exists(graph_name, output, access_token)
+    assert exists, output["logs"].append(f"ERROR: graph {graph_name} not found in grip")
+
+    output["logs"].append(f"loading files into {graph_name} from {directory_path}")
+
+    assert os.path.isdir(directory_path), output["logs"].append(f"directory path {directory_path} is not a directory")
```

Comment:
The etl pod captures logs through the output["logs"] structure so that when the user runs their job and it fails they have a list of logs. 

Print statements are also used in the etl pod, but these print statements are only visible to developers that have access to the cluster, so we should be replacing these print statements with the log ones

Reply:
okay sounds good, I'll look out for this in my testing